lambda_layers/core_layer/python/core_layer/handler/user_handler.py

```diff
@@ -160,7 +160,7 @@ def get_top_users_by_period(n, p, attr, descending, is_test, session) -> [User]:
     if session == None:
         session = get_db_session(is_test, session)
 
-    compare_timestamp = datetime.now() - timedelta(weeks=p)
+    compare_timestamp = helper.get_date_time(datetime.now(), is_test) - timedelta(weeks=p)
```

Comment:
It should look like this. `helper.get_date_time(datetime.now() - timedelta(weeks=p), is_test)`
Otherwise this will run into an error in production, because the function `helper.get_date_time` would  transform `datetime.now()` into a string, making in incompatible with `- timedelta(weeks=p)`

Reply:
got it, done!